### DISCUSSION TITLE
crashes when i try to change color

### DISCUSSION THREAD

**Discussion by analogVulpine** [Dpeta/pesterchum-alt-servers#174] [Q&A]
my computer is running Linux Mint 21.2 and i am trying to run Pesterchum Alt. v2.5.3 (glibc 2.31). it seems to be working fine, but whenever i try to change my color it crashes, and it won't display other people's colors properly, usually only doing it for 2 or 3.

**Comment by Dpeta** [comment]
Said this in the republic channel also but the color picker issue is probably because your desktop environment doesn't provide a color picker Qt can use, maybe try installing a package that provides one? Qt uses the native color picker by default so it differs per os/install. I'll try to add a fallback for it at some point.

Are you sure displaying colors incorrectly is a real issue? seems unrelated to the whole color picker thing

**Reply by analogVulpine** [reply]
i'm so sorry i completely forgot i made this post. trying that solution, it didn't work either. also tried it on mint 22.1. the terminal says it can't load gtk3. idk if the color issue is related. again, so sorry i forgot about this.
